docs/account/sso-login/okta-saml.md

```diff
@@ -70,6 +76,42 @@ Pleas be aware that, enabling SSO for **APPCIRCLE LOGIN** doesn't enable SSO for
 
 <Screenshot url='https://cdn.appcircle.io/docs/assets/oktasamlsettings3-new.png' />
 
+## Okta Managing User Groups
+
+- Login to your [Okta](https://www.okta.com/) account and navigate to Directory and then click ***Groups***
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2812-okta-groups-1.png' />
+
+- Click "Add group" button and after fill the fields click "Save" button
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2812-okta-groups-2.png' />
+
+- Click recently created group name and assign people to your group
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2812-okta-groups-3-new.png' />
+
+- Navigate to Applications tab and assign your application with "Assign application" button
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2812-okta-groups-4.png' />
+
+- Navigate to Applications and click integrated app
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2812-okta-groups-5.png' />
+
+- Navigate "General" tab and edit ***SAML Settings***
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2812-okta-groups-6.png' />
+
+- In Configure SAML step, you should set Group Attribute Statements
+- Name field should be matched with the ***CLAIM NAME (OPENID) / ATTRIBUTE NAME (SAML)*** field set in Appcircle. Name format should be `Basic`. 
+- If you want to get all the groups that you created you should set filter section as `Matches regex` and `.*`
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2812-okta-groups-7.png' />
+
+- Click on the "Next" button and finish the edit process.
+
```

Comment:
We might give an example use case here for the store.

For example there are two groups, one is `developers` and other one is `users`.

The beta channel should be available for `developers` group and not for `users` group.
The live channel should be available for both groups.